docs/package.json

```
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "3.6.3",
+    "@docusaurus/plugin-client-redirects": "^3.6.3",
```

```suggestion
    "@docusaurus/plugin-client-redirects": "3.6.3",
```
We've been keeping all the other docusaurus dependencies at fixed versions, let's make sure they all stay on the same version.

---

Good catch, thanks!